atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepDeleterImpl.java

```diff
@@ -52,7 +52,8 @@ public void sweep(Collection<WriteInfo> writes) {
         Map<Cell, Long> deleteTimestampByCell = Maps.newHashMap();
 
         for (WriteInfo write : writes) {
-            deleteTimestampByCell.merge(write.cell(), write.timestampToDeleteAtExclusive(sweeper), Long::max);
+            // todo(gmaretic): since we don't know about deletes, makes it awkward for thorough sweep?
```

Comment:
We should know about them if we plumb the isTombstone flag through on the `WriteInfo`s.

Reply:
This was added before our discussion so that I don't forget :)